Build Error: 'const_fn' feature removed

I get this error when running `cargo build`:

```
   Compiling libudis86-sys v0.2.1
   Compiling mmap-fixed v0.1.5
   Compiling detour v0.8.0
error[E0557]: feature has been removed
 --> E:\Users\justin\.cargo\registry\src\github.com-1ecc6299db9ec823\detour-0.8.0\src\lib.rs:4:11
  |
4 |   feature(const_fn, const_fn_trait_bound, unboxed_closures, abi_thiscall)
  |           ^^^^^^^^ feature has been removed
  |
  = note: split into finer-grained feature gates

error: aborting due to previous error

For more information about this error, try `rustc --explain E0557`.
error: could not compile `detour`

To learn more, run the command again with --verbose.
```

I'm running Windows and have tried both 32-bit targets, `i686-pc-windows-gnu` and `i686-pc-windows-msvc`

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -1,7 +1,7 @@
 #![recursion_limit = "1024"]
 #![cfg_attr(
   feature = "nightly",
-  feature(const_fn, const_fn_trait_bound, unboxed_closures, abi_thiscall)
+  feature(const_fn_trait_bound, unboxed_closures, abi_thiscall)
 )]
 #![cfg_attr(
   all(feature = "nightly", test),

